plans/dht/README.md

```diff
@@ -72,3 +79,23 @@ IPFS can safely rely on the latest DHT upgrades by running go-libp2p DHT tests d
     - Nodes ran 5 random-walk queries to populate their Routing Tables
   - **Act I**
     - Each node calls Provide for `i-provides` until it reaches a total of `n-provides`
+
+### `Test:` Provider record saturation
+
+> The libp2p DHT activates Load Balancing techniques once it understand that a node is owning a popular segment of the address space and with that, storing a larger amount of records that what it can handle. This test is designed to verify that libp2p is actively mitigating that by distributing that load with other nodes.
+
+We need to test:
+- Records are replicated to more nodes in the edges of the N closest nodes, once those nodes store a large amount of records (beyond what their computing resources enable)
+- Same as above, but in the inverse direction, that is: records are not replicated to more nodes, once the load is reduced
+- Records are replicated to more nodes in the edges of the N closest nodes, once those records are heavily requested (beyond what their bandwidth enables the N closest nodes to serve them)
+- Same as above, but in the inverse direction, that is: records are not replicated to more nodes, once the load is reduced
+
+### `Test:` Switch modes (client, full)

```

Comment:
This test will require simulated NATs. Will add a `blocked` until that is achievable